apis/python/tests/test_dense_nd_array.py

```diff
@@ -30,16 +23,16 @@ def test_dense_nd_array_create_ok(
     """
     assert pa.types.is_primitive(element_type)  # sanity check incoming params
 
-    a = soma.DenseNDArray(uri=tmp_path.as_posix())
     with pytest.raises(TypeError):
-        a.create_legacy(element_type.to_pandas_dtype(), shape)  # XXX
-    a.create_legacy(element_type, shape)
+        soma.DenseNDArray.create(
+            tmp_path.as_posix(), type=element_type.to_pandas_dtype(), shape=shape
+        )  # XXX
```

Comment:
what is `XXX` a reminder for?

Reply:
Not sure; [it was there when I got here](https://github.com/single-cell-data/TileDB-SOMA/blame/da1456d72fdcc1619a7139bdb45c640a0010a72d/apis/python/tests/test_dense_nd_array.py).